pyth/plugins/rtf15/reader.py

```diff
@@ -312,13 +328,14 @@ def handle_Para(self, para):
         prevListLevel = self.listLevel
         self.listLevel = para.listLevel
 
-        if self.listLevel > prevListLevel:
-            l = document.List()
-            self.listStack.append(l)
+        if prevListLevel is not None and self.listLevel is not None:
+            if self.listLevel > prevListLevel:
+                l = document.List()
+                self.listStack.append(l)
 
-        elif self.listLevel < prevListLevel:
-            l = self.listStack.pop()
-            self.listStack[-1].append(l)
+            elif self.listLevel < prevListLevel:
```

Comment:
I never tested item list handling; XHTMLWriter does not appear to do any.
It was just my local impression how the logic ought to work.